packages/calcite-components/stencil.config.ts

```diff
@@ -77,7 +77,7 @@ export const create: () => Config = () => ({
     { components: ["calcite-switch"] },
     { components: ["calcite-tab", "calcite-tab-title", "calcite-tab-nav", "calcite-tabs"] },
     { components: ["calcite-text-area"] },
-    { components: ["calcite-tile"] },
+    { components: ["calcite-tile-group", "calcite-tile"] },
```

Comment:
I think these need to be separate since tile can still be used standalone, right?

Reply:
k just updated, they are separate now.